src/Github/JwageGenerateChangelog.php

```diff
@@ -20,23 +21,26 @@
 final class JwageGenerateChangelog implements GenerateChangelog
 {
     private ChangelogGenerator $changelogGenerator;
+    private GitHubCredentials $gitHubCredentials;
 
-    public function __construct(ChangelogGenerator $changelogGenerator)
+    public function __construct(ChangelogGenerator $changelogGenerator, GitHubCredentials $gitHubCredentials)
     {
         $this->changelogGenerator = $changelogGenerator;
+        $this->gitHubCredentials  = $gitHubCredentials;
     }
 
     public static function create(
         RequestFactoryInterface $messageFactory,
-        ClientInterface $client
+        ClientInterface $client,
+        GitHubCredentials $gitHubCredentials
     ): self {
         $issueClient     = new IssueClient($messageFactory, $client);
         $issueFactory    = new IssueFactory();
         $issueFetcher    = new IssueFetcher($issueClient);
         $issueRepository = new IssueRepository($issueFetcher, $issueFactory);
         $issueGrouper    = new IssueGrouper();
 
-        return new self(new ChangelogGenerator($issueRepository, $issueGrouper));
+        return new self(new ChangelogGenerator($issueRepository, $issueGrouper), $gitHubCredentials);
```

Comment:
I think this code path is not tested, hence why the build fails: can it be covered with a new test that just verifies that a `JwageGenerateChangelog` is produced?

Reply:
Absolutely. Not at home right now but I’ll do it later, thanks!